src/context/wishlist-context.js

```diff
@@ -0,0 +1,79 @@
+import axios from "axios";
+import { createContext, useContext, useState, useEffect } from "react";
+import { useAuth } from "./auth-context";
+
+const WishlistContext = createContext();
+
+const WishlistProvider = ({ children }) => {
+  const { authData } = useAuth();
+  const [wishlist, setWishlist] = useState([]);
+
+  useEffect(() => {
+    if (authData.isLoggedIn) {
+      (async () => {
+        try {
+          const { data, status } = await axios.get(`/api/user/wishlist`, {
+            headers: { authorization: localStorage.getItem("token") },
+          });
+          if (status === 200) {
+            setWishlist(data.wishlist);
+          }
+        } catch (e) {
+          console.log("Error in Getting Wishlist", e);
```

Comment:
For these kinds of messages, you can use: `console.error()`

Reply:
Done